### PR TITLE
Feat/cypress log in dashboard

### PR DESCRIPTION
**Pull request by LeftyLincoln**
# Description

-Cypress Testing for Login Page
-Cypress Testing for Dashboard View

# Reviewer Questions

-Any other testing you see needed currently?

## Type Of Change

- [ ]  Bug fix
- [ ]  New feature
- [ ]  Refactoring
- [ X]  Testing

# How Has This Been Tested?

- [ ]  local host
- [ ]  dev tools
- [X ]  cypress

# Related Issues:

- [ ]  closes  

# Checklist:

- [x]  My code follows the Turing's guidelines of this project
- [x]  I have performed a self-review of my own code
- [x]  No console error messages

### CODE REVIEW

File: cypress/e2e/dashboard.cy.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    cy.get('.chore-container > :nth-child(2)').contains("120 minutes")
+  })
+  it('should show a message if no chores are given for the day', () => {
+    cy.get('.Tuesday').click()
+    cy.get('.day-off').contains("You have the day off, no chores today!")
+  })
+
+  it('should allow a user to log out and go back to the login page', () => {
+    cy.get('.active > .nav-btn').click()
+    cy.url().should('eq', "https://to-do-guru-ui.vercel.app/")
```
Comment (Kanderson58) [inline cypress/e2e/dashboard.cy.js:39]: Good catch on that one @qrispi

File: cypress/e2e/dashboard.cy.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  })
+
+  it('should show allow a user to edit the household by bringing them back the household form page', () => {
+    cy.get('[href="/houseform"] > .nav-btn').click()
+    cy.url().should('eq', "https://to-do-guru-ui.vercel.app/houseform")
+  })
+
+  it('should show allow a user to edit the chores by bringing them back the chore form page', () => {
+    cy.get('[href="/choreform"] > .nav-btn').click()
+    cy.url().should('eq', "https://to-do-guru-ui.vercel.app/choreform")
```
Comment (Kanderson58) [inline cypress/e2e/dashboard.cy.js:49]: Might be redundant but I've gotten into the habit of checking that actual JSX elements no longer exist/do exist when I change pages to make sure that router is properly rendering what we need it to

**Review by Kanderson58 — COMMENTED** [review]
Great work, thanks for handling this!

**Review by qrispi — APPROVED** [review]
LGTM!

**Pull request merged**